### PR TITLE
Add plotly descriptors

### PR DESCRIPTION
**Pull request by altavir**
Plotly.kt and plotly.kt-server descriptors accompanied by warning message about glitches.

### CODE REVIEW

**Comment by ileasile** [comment]
Thank you for providing descriptors!
Please add `description` property to these JSONs: https://github.com/Kotlin/kotlin-jupyter#adding-new-libraries
And then regenerate README using `./gradlew generateReadme` command.

File: libraries/plotly-server.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "imports": [
+        "kscience.plotly.*",
+        "kscience.plotly.models.*",
+        "kscience.plotly.server.JupyterPlotlyServer",
+        "hep.dataforge.meta.*",
+        "kotlinx.html.*"
+    ],
+    "repositories": [
+        "*mavenLocal",
```
Comment (ileasile) [inline libraries/plotly-server.json:10]: Don't you mind removing this?

File: libraries/plotly.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "imports": [
+        "kscience.plotly.*",
+        "kscience.plotly.models.*",
+        "kscience.plotly.JupyterPlotly",
+        "hep.dataforge.meta.*",
+        "kotlinx.html.*"
+    ],
+    "repositories": [
+        "*mavenLocal",
```
Comment (ileasile) [inline libraries/plotly.json:10]: The same thing here

File: libraries/plotly-server.json (hunk 1 above)
Comment (altavir) [reply libraries/plotly-server.json]: done

File: libraries/plotly.json (hunk 2 above)
Comment (altavir) [reply libraries/plotly.json]: done

**Comment by altavir** [comment]
Applied both changes. Additionally fixe kmath descriptor.

**Pull request merged**